README.md

```diff
@@ -2,7 +2,7 @@
 
 <div align="center">
   <!-- You are encouraged to replace this logo with your own! Otherwise you can also remove it. -->
-  <img src="murple_logo.png" alt="logo" width="140"  height="auto" />
+  <img src="./murple_logo.png" alt="logo" width="140"  height="auto" />
```

Comment:
- [ ] You are not expected to use the microverse logo in your readme, this should be your logo, however if you do not have one you are to remove this microverse logo 👍